operator/builtin/input/tcp/tcp.go

```diff
@@ -30,6 +30,7 @@ func NewTCPInputConfig(operatorID string) *TCPInputConfig {
 type TCPInputConfig struct {
 	helper.InputConfig `yaml:",inline"`
 
+	MaxBufferSize int       `json:"max_buffer_size,omitempty" yaml:"max_buffer_size,omitempty"`
```

Comment:
We should probably use `helper.ByteSize` here.

Reply:
The latest commit uses helper.ByteSize.